pkg/core/manager/insight/summary.go

```diff
@@ -205,3 +212,40 @@ func (i *InsightManager) GetResourceGroupSummary(ctx context.Context, client *mu
 		CountByGVK:    topFiveCount,
 	}, nil
 }
+
+// GetPodStatus returns the status of a pod
+func GetPodStatus(obj map[string]any) string {
+	containerStatuses, found, err := unstructured.NestedSlice(obj, "status", "containerStatuses")
+	if err != nil || !found || len(containerStatuses) == 0 {
+		return "Unknown"
+	}
+	firstContainer, ok := containerStatuses[0].(map[string]any)
+	if !ok {
+		return "Unknown"
+	}
+	state, found := firstContainer["state"]
+	if !found {
+		return "Unknown"
+	}
+	stateMap, ok := state.(map[string]interface{})
+	if !ok {
+		return "Unknown"
+	}
+	if stateMap["running"] != nil {
+		return "Running"
+	}
+	if stateMap["waiting"] != nil {
```

Comment:
If the container is in `waiting` state, print the detail `reason`, e.g. `ErrImagePull`, `ClashLoopBackOff`